### DISCUSSION TITLE
Is it possible to export the error-prone checks result as some kind of report?

### DISCUSSION THREAD

**Discussion by faisal6621** [google/error-prone#4458] [Q&A] [answered]
Currently all the warning and errors are reported in logs. But could it be possible to generate an aggregated report where we can easily manage them? Like group all related items at once, filter out based on severity, etc.
The report could be HTML. CSV or some other format.

**Comment by tbroyer** [comment, marked as answer]
See #3766 and #444 

TL;DR: it's not possible currently (and it's not clear whether ErrorProne should support it vs. JavaC itself or some other more generic tool).
